mne/coreg.py

```diff
@@ -1245,3 +1248,632 @@ def _scale_xfm(subject_to, xfm_fname, mri_name, subject_from, scale,
+        self._scale_mode = None
+
+        self._rot_trans = None
+        self._default_parameters = \
+            np.array([0., 0., 0., 0., 0., 0., 1., 1., 1.])
+
+        self._icp_iterations = 20
+        self._icp_angle = 0.2
+        self._icp_distance = 0.2
+        self._icp_scale = 0.2
+        self._icp_fid_matches = ('nearest', 'matched')
+        self._icp_fid_match = self._icp_fid_matches[0]
+        self._lpa_weight = 1.
+        self._nasion_weight = 10.
+        self._rpa_weight = 1.
+        self._hsp_weight = 1.
+        self._eeg_weight = 1.
+        self._hpi_weight = 1.
+
+        self._extra_points_filter = None
+        self._dig = _get_data_as_dict_from_dig(
+            dig=self._info['dig'],
+            exclude_ref_channel=False
+        )
+        # adjustments
+        self._dig['rpa'] = np.array([self._dig['rpa']])
+        self._dig['nasion'] = np.array([self._dig['nasion']])
+        self._dig['lpa'] = np.array([self._dig['lpa']])
+
+        self._setup_mri(subjects_dir=self._subjects_dir,
+                        subject=self._subject)
+        self.reset()
+        self._update_params(
+            rot=self._default_parameters[:3],
+            tra=self._default_parameters[3:6],
+            sca=self._default_parameters[6:9],
+        )
+
+    def _setup_mri(self, subjects_dir, subject):
+        # find high-res head model (if possible)
+        high_res_path = _find_head_bem(subject, subjects_dir, high_res=True)
+        low_res_path = _find_head_bem(subject, subjects_dir, high_res=False)
+        if high_res_path is None and low_res_path is None:
+            raise RuntimeError("No standard head model was "
+                               f"found for subject {subject}")
+        if high_res_path is not None:
+            self.bem_high_res = self._read_surface(high_res_path)
+        else:
+            self.bem_high_res = self._read_surface(low_res_path)
+        if low_res_path is None:
+            # This should be very rare!
+            warn('No low-resolution head found, decimating high resolution '
+                 'mesh (%d vertices): %s' % (len(self.bem_high_res.surf.rr),
+                                             high_res_path,))
+            # Create one from the high res one, which we know we have
+            rr, tris = decimate_surface(self.bem_high_res.surf.rr,
+                                        self.bem_high_res.surf.tris,
+                                        n_triangles=5120)
+            # directly set the attributes of bem_low_res
+            self.bem_low_res = complete_surface_info(dict(rr=rr, tris=tris),
+                                                     copy=False, verbose=False)
+        else:
+            self.bem_low_res = self._read_surface(low_res_path)
+
+        # Set MNI points
+        try:
+            fids = get_mni_fiducials(subject, subjects_dir)
+        except Exception:  # some problem, leave at origin
+            self.mni_points = None
```

Comment:
Are you actually testing this? I fear it cannot work in this case. I would consider a pure crash and allow users to specify ‘fids’ while if set to ‘auto’ we get is from MNI fsaverage.

Reply:
I tried 235f8b89fc1447a0806e7bf688922d5998bd4069 but I'm not 100% sure this is what you mean, feel free to take a look